app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -44,7 +45,10 @@ data class OverdueAppointment(
   @Dao
   interface RoomDao {
 
-    @Query("""
+    companion object {
+
+      @Language("RoomSql")
+      private const val OVERDUE_APPOINTMENTS_QUERY = """
```

Comment:
Since we moved `WHERE`, `GROUP BY` and `ORDER BY` to const. We cannot add new conditions like search later on. Ideally, the const query should only contain the query without the conditions, so that we can define them per query function basis. We can get on a short call if you want more information about this issue.

Reply:
I got it, thanks for pointing this out 👍🏻